docs/abci-cli.rst

```diff
@@ -35,17 +35,21 @@ Now run ``abci-cli`` to see the list of commands:
 
     Available Commands:
       batch       Run a batch of abci commands against an application
+      begin_block Delineate beginning of a block with a header and a hash
```

Comment:
https://github.com/tendermint/abci/pull/73 not yet merged

Reply:
Yes. I changed the description to say `waits on`. Once the tendermint/abci#73 gets merged this can get merged too.